packages/create-turbo/src/cli.ts

```diff
@@ -21,9 +21,17 @@ const createTurboCli = new Command();
 createTurboCli
   .name(chalk.bold(logger.turboGradient("create-turbo")))
   .description("Create a new Turborepo")
-  .usage(`${chalk.bold("<project-directory> <package-manager>")} [options]`)
+  .usage(`${chalk.bold("<project-directory>")} [options]`)
   .argument("[project-directory]")
+  // TODO: argument is still provided (but removed from help)
+  // for backwards compatibility, remove this in the next major
   .argument("[package-manager]")
+  .addOption(
+    new Option(
+      "-m, --manager <package-manager>",
```

Comment:
Making this `--package-manager` for clarity. Short flag exists for frequent users.